Mathlib/RepresentationTheory/Homological/GroupHomology/Functoriality.lean

```diff
@@ -317,6 +322,134 @@ lemma H1π_comp_map :
     H1π A ≫ map f φ 1 = mapCycles₁ f φ ≫ H1π B := by
   simp [H1π, Iso.inv_comp_eq, ← cyclesMap_comp_isoCycles₁_hom_assoc]
 
+@[simp]
+lemma map_1_one (φ : A ⟶ (Action.res _ (1 : G →* H)).obj B) :
```

Comment:
oh I'm not sure I like 1 in a lemma name to mean 1, we have `one` for that. What's wrong with `map_one_one`?

Reply:
Thanks and sorry for the slow response. `map₁_one` seems more consistent with the convention of referring to stuff about `n`th homology with subscript `n`, even though `map₁` isn't a definition. `groupHomology.cyclesIso₀` is a similar case.

Reply:
Yeah I see now, this is fine.